### PR TITLE
fix: mobile ui fixes

### PR DESCRIPTION
**Pull request by richardshiue**
1. adjust the padding in mobile grid, 16px on either side
2. reordering fields on mobile not working

### Feature Preview

---

#### PR Checklist

- [ ] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/mobile_grid_page.dart [hunk 1]
```diff
@@ -216,17 +219,22 @@ class _GridPageContentState extends State<GridPageContent> {
...
 
   @override
   Widget build(BuildContext context) {
     return BlocBuilder<GridBloc, GridState>(
       builder: (context, state) {
         return MobileGridHeader(
           viewId: state.viewId,
-          anchorScrollController: headerScrollController,
+          scrollController1: controller1,
+          scrollController2: controller2,
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database/grid/presentation/mobile_grid_page.dart:237]: These names are bad..

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart [hunk 2]
```diff
@@ -41,29 +46,45 @@ class _MobileGridHeaderState extends State<MobileGridHeader> {
...
+          BlocBuilder<GridHeaderBloc, GridHeaderState>(
+            builder: (context, state) {
+              return SingleChildScrollView(
+                scrollDirection: Axis.horizontal,
+                controller: widget.scrollController1,
+                child: Stack(
+                  children: [
+                    Positioned(
+                      top: 0,
+                      left: GridSize.leadingHeaderPadding + 24,
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart:60]: We're using this for a lot of different stuff, should it still be named `leadingHeaderPadding`?

Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart:60]: It implies the same metric here. The amount of space between the edge and the first element in the database view header or the database view content

Comment (Xazin) [reply frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart:60]: Leading means it's only before, we use it all around now.

`GridSize.horizontalHeaderPadding`

Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart:60]: Oh I get what you mean

**Review by Xazin — APPROVED** [review]
LGTM (with the few variable name changes)

**Pull request merged**